portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationDisabledAtInstance.testcase

```diff
@@ -31,87 +31,31 @@ definition {
-				locator1 = "FormFields#FIELD_DISABLED_TEXT");
-		}
-	}
-
-	@description = "LPS-153084 Given the 'Enable Assign Roles by Segment' is disable at Instance Settings And enabled at System Settings
-    When user navigates to Regular Roles > Assignees > Segments
-    Then the warning alert explains how to re-enable it"
+	@description = "LPS-152539: Given segmentation is disabled at Instance Settings
+	And user has permissions to Instance Settings
+	When user clicks at breadcrumb's link at Dynamic Collection editor
+	Then should be forwarded to Segments Service at Instance Settings "
 	@priority = "3"
-	test CheckRoleDisabledAlertAtInstanceSettings {
-		property test.name.skip.portal.instance = "SegmentationDisabled#CheckRoleDisabledAlertAtInstanceSettings";
-
-		task ("Disable asssigning role by segment at Instance Settings") {
-			Segmentation.disableSegmentationAtSystemSettings();
-		}
-
-		task ("Create a Role") {
-			JSONRole.addRegularRole(
-				roleKey = "Test Role",
-				roleTitle = "Test Role");
-		}
-
-		task ("Navigate to Role>Assignee>Segments") {
-			Segmentation.gotoRoleAssigneeSegments(roleName = "Test Role");
-		}
-
-		task ("Check alert message") {
-			Segmentation.checkSegmentationDisabledWarning(textValue = "Assigning roles by segment is disabled. To enable, go to Instance Settings.");
-		}
-
-		task ("Reset System Settings configuration") {
-			Segmentation.resetSegmentationAtSystemSettings();
+	test ClickToReEnableAlertLinkAtDynamicCollectionEditor {
+		task ("Create a Collection via JSONWS") {
+			JSONAssetlist.addDynamicAssetListEntry(
+				groupName = "Test Site Name",
+				title = "Dynamic Collection Test");
 		}
-	}
 
-	@description = "LPS-152541 Given the 'Enable Segmentation' is disabled at System Settings
-    When user navigates to Segmentation at Instance Settings
-    Then should see an alert informing that is disabled and how to re-enable it"
-	@ignore = "true"
-	@priority = "4"
-	test CheckSegmentationDisabledAlertAtInstanceSettings {
-		task ("No alert when Segmentation Enabled") {
-			Segmentation.gotoSegmeentsService(key_portlet = "Instance Settings");
-
-			AssertElementNotPresent(locator1 = "Message#ERROR_3");
-		}
+		task ("Navigate to Collection Admin") {
+			AssetListsAdmin.openAssetListsAdmin(siteURLKey = "test-site-name");
 
-		task ("Disable Segmentation at Instance Settings") {
-			Segmentation.disableConfigurationAtSystemSettings(checkLabel = "Enable Segmentation");
+			AssetListsAdmin.editAssetList(assetListTitle = "Dynamic Collection Test");
 		}
 
-		task ("Check alert when Segmentation Disabled") {
-			Segmentation.gotoSegmeentsService(key_portlet = "Instance Settings");
-
-			Segmentation.checkSegmentationDisabledWarning(textValue = "Segmentation is disabled in System Settings. To enable, go to System Settings.");
+		task ("Click at warning with link and confirm Instance Settings is open") {
+			Segmentation.clickToReEnableSegmentsWarningLink();
```

Comment:
This macro has the same AssertVisible function is being used at the testcase, so it's a good idea to remove it from there (or just leave the click). There are some other parts with this situation.

Reply:
ok! good idea, i will fix!